ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/execution/LedgerTimeAwareCommandExecutor.scala

```diff
@@ -95,16 +95,19 @@ final class LedgerTimeAwareCommandExecutor(
       }
   }
 
+  // Does nothing if `t` is empty. This can happen if the transaction only regarded divulged contracts.
```

Comment:
```suggestion
  // Does nothing if `newTime` is empty. This can happen if the transaction only regarded divulged contracts.
```

Reply:
I thought I fixed this, thanks.

Reply:
Uh, it was just committed locally. :sweat_smile: But good catch, thanks for reading the comments.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/5541/commits/759e5cacf0196ae492974383b47f1400667f1ac5

Reply:
Adddressed by https://github.com/digital-asset/daml/pull/5541/commits/abfc4cf353d52e71cce74396ea958554f7f06b30